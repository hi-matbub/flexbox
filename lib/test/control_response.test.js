import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import ControlResponse from '../comps/coming-soon_.jsx';

it('renders smoke test without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<ControlResponse />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders snapshot as expected', () => {
  const tree = renderer.create(<ControlResponse />).toJSON();
  expect(tree).toMatchSnapshot();
});
