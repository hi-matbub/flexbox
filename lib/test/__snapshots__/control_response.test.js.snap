// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot as expected 1`] = `
Array [
  <h1
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    It's almost here! Be the first to know
  </h1>,
  <div
    id="mc_embed_signup"
  >
    <form
      action="https://matbub.us17.list-manage.com/subscribe/post?u=eb89624098c8ce40b7f3b26ef&id=909f807d9f"
      className="validate"
      id="mc-embedded-subscribe-form"
      method="post"
      name="mc-embedded-subscribe-form"
      noValidate={true}
      target="_blank"
    >
      <div
        className="form-cont"
      >
        <input
          aria-label="Repository description"
          className="email"
          defaultValue=""
          id="mce-EMAIL"
          name="EMAIL"
          placeholder="e.g. hi@matbub.co"
          style={
            Object {
              "border": 0,
            }
          }
          type="email"
        />
        <div
          className="clear"
          id="mce-responses"
        >
          <div
            className="response"
            id="mce-error-response"
            style={
              Object {
                "display": "none",
              }
            }
          />
          <div
            className="response"
            id="mce-success-response"
            style={
              Object {
                "display": "none",
              }
            }
          />
        </div>
        <button
          className="button"
          id="mc-embedded-subscribe"
          name="subscribe"
          type="submit"
        >
          Subscribe
        </button>
      </div>
    </form>
  </div>,
]
`;
